Dockerfile

```diff
@@ -13,15 +11,16 @@ ENV HOME /home/lounge
 ENV LOUNGE_HOME ${HOME}/data
 ENV LOUNGE_SRC ${HOME}/src
 
-RUN mkdir -p ${LOUNGE_SRC}
-WORKDIR ${LOUNGE_SRC}
-
-RUN curl -L https://github.com/thelounge/lounge/archive/v${LOUNGE_VERSION}.tar.gz | tar --strip-components 1 -xzf -
-
-RUN chown -R lounge:lounge ${LOUNGE_SRC}
+# Create a non-root user for lounge to run in.
+RUN useradd --create-home --uid 1000 lounge
 
 # Drop root.
 USER lounge
+
+RUN mkdir -p ${LOUNGE_SRC}
+WORKDIR ${LOUNGE_SRC}
+
+RUN curl -L $(npm view thelounge@${LOUNGE_VERSION} dist.tarball) | tar --strip-components 1 -xzf -
```

Comment:
> This would also get rid of LOUNGE_VERSION, thus require less maintenance.

I don't see why one would want to not version the bundled lounge version though?

> And instead of using NODE_ENV, you could pass in --only=prod.

`NODE_ENV` does more than just tell npm to install production only, it's the de-facto environment variable for Node apps. I know Express uses it internally for stuff, as well as many other modules (lounge probably has uses for it as well).